src/client/binary_client.cpp

```diff
@@ -375,7 +375,7 @@ namespace
       request.MethodsToCall = methodsToCall;
       const CallResponse response = Send<CallResponse>(request);
       if (Debug) {std::cout << "binary_clinent | Call <--" << std::endl;}
-      return response.Results;
+      return response.Parameters.Results;
```

Comment:
exactly!

On Sun, 31 May 2015 at 13:39 Denis Štogl <notifications@github.com> wrote:

> In src/client/binary_client.cpp
> <https://github.com/FreeOpcUa/freeopcua/pull/143#discussion_r31389370>:
>
> > @@ -375,7 +375,7 @@ namespace
> >        request.MethodsToCall = methodsToCall;
> >        const CallResponse response = Send<CallResponse>(request);
> >        if (Debug) {std::cout << "binary_clinent | Call <--" << std::endl;}
> > -      return response.Results;
> > +      return response.Parameters.Results;
>
> I would actually like to not to split the structure since, it will be
> easier comparable with Specification (Part 4, page 62).
>
> But how to do this? Should the "CallResult" be added in 'NoSplitStruct'
> list?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeOpcUa/freeopcua/pull/143/files#r31389370>.
>